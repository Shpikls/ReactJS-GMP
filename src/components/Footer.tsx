import * as React from 'react'
import styled from 'styled-components'
import { Logo } from './Logo'

const Footer = (): JSX.Element => {
	return (
		<FooterWrapper>
			<Logo />
		</FooterWrapper>
	)
}

export { Footer }

const FooterWrapper = styled.footer`
	background: #424242;
	width: 1200px;
	margin: 0 auto;
	padding: 20px 56px;
	box-sizing: border-box;
	color: #ffffff;
	display: flex;
	justify-content: center;
`
