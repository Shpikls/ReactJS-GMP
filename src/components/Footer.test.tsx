import { render } from '@testing-library/react'
import React from 'react'
import { Footer } from './Footer'

describe('Testing Footer.tsx', () => {
	test('To mach snapshot', () => {
		const { asFragment } = render(<Footer />)
		expect(asFragment()).toMatchSnapshot()
	})
})
