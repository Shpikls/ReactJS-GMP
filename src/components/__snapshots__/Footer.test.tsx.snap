// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer.tsx To mach snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-gsTEea ZreqD"
  >
    <span
      class="sc-bdfBQB iqkcpy"
    >
      <span>
        netflix
      </span>
      roulette
    </span>
  </footer>
</DocumentFragment>
`;
